app/controllers/partners_controller.rb

```diff
@@ -15,11 +15,15 @@ def index
   end
 
   def create
-    @partner = current_organization.partners.new(partner_params)
-    if @partner.save
-      redirect_to partners_path, notice: "Partner added!"
+    svc = PartnerCreateService.new(organization: current_organization, partner_attrs: partner_params)
```

Comment:
What's `svc`? A service or a service creator?

Reply:
@scooter-dangle `svc` would be instance of the service objective. The service creator would be `PartnerCreateServce.new`. Hope that makes sense. Always looking to make things clearer